main.tf

```diff
@@ -369,7 +371,7 @@ resource "ibm_resource_instance" "orchestrate_instance" {
 
   lifecycle {
     precondition {
-      condition     = contains(["us-south"], var.location)
+      condition     = contains(["us-south"], var.region)
       error_message = "watsonx Orchestrate is only available in us-south region."
```

Comment:
This is wrong. watsonx Orchestrate is supported in various regions. [Reference](https://github.com/terraform-ibm-modules/terraform-ibm-watsonx-orchestrate/blob/9652d540793ace18a2634f96efb70ae29794fa01/variables.tf#L29)

<img width="751" height="652" alt="Image" src="https://github.com/user-attachments/assets/23740e73-2034-4a16-8af1-a9248c3835b1" />